pkg/cluster/nodes/node.go

```diff
@@ -279,6 +286,28 @@ func (n *Node) IP() (ip string, err error) {
 	return ip, nil
 }
 
+// IPPrefix returns the Subnet prefix of the node
+func (n *Node) IPPrefix() (ipPrefix string, err error) {
```

Comment:
The docker package should implement IPPrefix, at some point the nodes package should only use a shim interface to {docker,podman,a fake...}